### PR TITLE
Removes punch button and adds dropdown options on menu

### PR DESCRIPTION
**Pull request by LucasHFS**
solves #15.

cm42

### CODE REVIEW

File: app/views/refills/_navigation.html.erb [hunk 1]
```diff
@@ -1,29 +1,44 @@
 <header>
   <div>
-    <nav class="navbar navbar-expand navbar-dark static-top">
-      <div class="navbar-brand">
+    <nav class='navbar navbar-expand navbar-dark static-top'>
```
Comment (wenderjean) [inline app/views/refills/_navigation.html.erb:3]: For the sake of our sanity, I believe we should continue with double quotes.

**Review by wenderjean — CHANGES_REQUESTED** [review]

File: app/views/refills/_navigation.html.erb [hunk 2]
```diff
@@ -9,11 +9,26 @@
...
+                <%= link_to dashboard_index_path, class:"justify-content-start" do %>
+                  <%= t('.multiple_punches') %>
+                <%- end %>
+              </li>
+            </ul>
+          </div>
+
           <% if can? :read, Evaluation %>
-            <%= link_to t('activerecord.models.evaluation.other'), evaluations_path %>
+            <%= link_to t('activerecord.modelsevaluation.other'), evaluations_path %>
```
Comment (wenderjean) [inline app/views/refills/_navigation.html.erb:31]: Is this change intentional?

File: config/locales/pt-BR/pt-BR.yml [hunk 3]
```diff
@@ -51,7 +51,9 @@ pt-BR:
   refills:
     navigation:
       logout: "Sair"
-      dashboard: 'Dashboard'
+      punches: 'Punches'
```
Comment (wenderjean) [inline config/locales/pt-BR/pt-BR.yml:54]: Can we continue using double quotes here also to keep the file consistent?

File: config/locales/pt-BR/pt-BR.yml [hunk 4]
```diff
@@ -51,7 +51,9 @@ pt-BR:
   refills:
     navigation:
       logout: "Sair"
-      dashboard: 'Dashboard'
+      punches: "Punches"
+      single_punch: "Single Punch"
```
Comment (wenderjean) [inline config/locales/pt-BR/pt-BR.yml:55]: Considerando que estamos tratando do arquivo `pt-BR.yml`, vai ser mais interessante se colocarmos algo como:
```
punches: "Punches"
single_punch: "Cadastrar Punch"
multiple_punches: "Múltiplos Punches"
```

Esse versão atual leve para o `en.yml` por favor.

**Comment by adbatista** [comment]
please rebase your code to sync with the new master chantes

**Review by wenderjean — APPROVED** [review]

**Pull request merged**